run.py

```
@@ -0,0 +1,813 @@
+            r.raise_for_status()
+            info = r.json()
+            if info is None or not info["data"]:
+                status = TwitchResponseStatus.OFFLINE
+            else:
+                status = TwitchResponseStatus.ONLINE
+       except requests.exceptions.RequestException as e:
+            if e.response:
+                if e.response.status_code == 401:
+                    status = TwitchResponseStatus.UNAUTHORIZED
+                if e.response.status_code == 404:
+                    status = TwitchResponseStatus.NOT_FOUND
```

Ruff (0.8.2)

Local variable `status` is assigned to but never used

Remove assignment to unused variable `status`

(F841)